modules/consentManagement.js

```diff
@@ -465,6 +465,9 @@ export function setConsentConfig(config) {
     utils.logInfo(`consentManagement config did not specify allowAuctionWithoutConsent.  Using system default setting (${DEFAULT_ALLOW_AUCTION_WO_CONSENT}).`);
   }
 
+  // if true, then gdprApplies should be set to true
+  gdprScope = config.defaultGdprScope === true;
```

Comment:
Is there a use-case where the pub could set the `defaultGdprScope` to `false`?  If so, should we carry that through the module's logic, so that the `gdprApplies` flag would have a back-up value of `false`?

Reply:
Disregard the above question.